DataFormats/HcalDigi/bin/BuildFile.xml

```diff
@@ -0,0 +1,11 @@
+<architecture name="slc6_amd64">
+    <flags LDFLAGS="-Wl,-dynamic-linker,$(GLIBC_BASE)/lib64/ld.so"/>
```

Comment:
This does not seem like a good idea - if you need something like this, it is probably a symptom of some other problem ?

Reply:
that was copied from the BuildFile.xml for cmsRun.cpp, not more than that